Exiled.Events/Patches/Fixes/OverwatchSkipFix.cs

```diff
@@ -22,6 +22,7 @@ namespace Exiled.Events.Patches.Fixes
 
     /// <summary>
     /// Fixes <see cref="CharacterClassManager.RunDefaultClassPicker(bool, out int[], out System.Collections.Generic.Dictionary{UnityEngine.GameObject, RoleType})"/> method.
```

Comment:
The top line still needs to be changed. `Patches <see cref="CharacterClassManager.RunDefaultClassPicker(bool, out int[], out System.Collections.Generic.Dictionary{UnityEngine.GameObject, RoleType})"/>.` should work.

Reply:
oof, i fixed again